### ISSUE TITLE
Ruby implementation

### ISSUE THREAD

**Issue by branch14** [SURFnet/ocra-implementations#1]
FYI I implemented OCRA in Ruby.

see https://github.com/branch14/rocra

Just in case you might want to link to it form the README of your project.

Best phil

**Comment by ijansch** [comment]
Nice! Would it be an idea to add it to the ocra-implementations repo instead of a separate one?

**Comment by branch14** [comment]
Yes, I thought about that. But on the other hand I wanted to have ruby gem which I can use in a project of mine. So I decided to have my own repo.

Nevertheless, feel free to copy my code into your repo if you wish.
